README.md

```diff
@@ -126,57 +126,34 @@ Work on FDC3 is split into several working groups and releases.
-
-Upon enrollment as an [FDC3 voting participant](https://github.com/orgs/finos/teams/fdc3-participants), you will be invited to join the [FINOS GitHub organization](https://github.com/orgs/finos/people) and the [fdc3-participants](https://github.com/orgs/finos/teams/fdc3-participants) GitHub team.
+If you or your firm are new to contributing to open source projects, please see the variety of resources available from FINOS, (such as the [Open Source readiness project](https://www.finos.org/open-source-readiness)), Linux Foundation ([Participating in Open Source communities](https://www.linuxfoundation.org/tools/participating-in-open-source-communities/)) and others (e.g. [opensource.guide](https://opensource.guide/)).
 
-#### How do you become an Editor or Maintainer?
+### What roles do contributors hold?
 
-Once you are an enrolled participant in FDC3, you can apply to become an **editor** or **maintainer** by contacting the existing FDC3 maintainers at [fdc3-maintainers@finos.org](mailto:fdc3-maintainers@finos.org) and then seeking the approval of the FDC3 Standards Working Group. Generally, the maintainers will look for both a history of contribution to FDC3 and a commitment to investing sufficient time in the role from any prospective candidates before proposing them to the Standards Working Group for approval. If you are new to FDC3, but willing to make the investment of time, the maintainers can work with you to build up a history of contribution.
+Please refer to the FDC3 [Governance document](GOVERNANCE.md) for details of the different roles on the FDC3 project.
 
 ### FINOS Code of Conduct
 
-Participants in FINOS standards projects should follow the FINOS Code of Conduct, which can be found at: <https://community.finos.org/docs/governance/code-of-conduct>
+Contributors to FINOS standards projects should follow the FINOS Code of Conduct, which can be found at: <https://community.finos.org/docs/governance/code-of-conduct>
```

Comment:
@robmoffat - why using the term "Contributors" here? Shouldn't we stick to the roles defined in Governance.md , to avoid confusion?

Reply:
This is a good question and I'm interested to see what you think about it.   So, we don't define what a participant is in the README file, so I didn't feel it was the right place to introduce that term.  Also, the section is called "Contributing" in the README... so it felt more natural to continue to call it that.  However, if we want to avoid the noun "Contributor" we could say something like:

> When contributing to FINOS standards projects, you should follow... 

Would that be better do you think?

Reply:
Yes, I think your suggestion makes sense.

I would also suggest to replace `Please see the Contributing guide for details on how to contribute to FDC3.` with

`Please see the Governance and Contributing guides for details on how to contribute to FDC3`, as it would allow the reader to navigate to the roles definitions. WDYT?